recipes/cas-offinder/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: cas-offinder
+  version: 2.4
+
+source:
+  url: https://github.com/snugel/cas-offinder/archive/5a5d24922005db5ab47dc0aceb0572a00b8f0e27.tar.gz
+  sha256: 7946039383fb826618959b6c729c8322355db59e2303ebaf1f2883f0365c103c
+
+requirements:
+  build:
+    - cmake # [unix]
+    - pocl ==0.14 # [unix]
+    - libgcc # [linux]
+    - llvm # [osx]
+  run:
+    - cmake # [unix]
+    - pocl ==0.14 # [unix]
+    - libgcc # [linux]
+test:
+  commands:
+    - cas-offinder -h
+
+about:
+  home: https://github.com/snugel/cas-offinder
+  license: BSD
+  license_file: LICENSE
+  summary: "a package that finds genome-wide off-target sites by limited number of mismatches using CRISPR/Cas9 system"
```

Comment:
Maybe use the official description from https://github.com/snugel/cas-offinder/blame/5a5d24922005db5ab47dc0aceb0572a00b8f0e27/README.md#L4-L5?
```yaml
  summary: Cas-OFFinder is OpenCL based, ultrafast and versatile program that searches for potential off-target sites of CRISPR/Cas-derived RNA-guided endonucleases (RGEN).
```

Reply:
I have substitute the summary as suggested from official description. Thank you!